colabs/StatisticalTests.ipynb

```diff
@@ -0,0 +1,995 @@
+{
+ "nbformat": 4,
+ "nbformat_minor": 0,
+ "metadata": {
+  "colab": {
+   "provenance": [],
+   "collapsed_sections": [],
+   "toc_visible": true
+  },
+  "kernelspec": {
+   "name": "python3",
```

Comment:
Kolejna słaba wydajnościowo konwersja `print(kstest([(x - m_r) / sd_r for x in ratings], "norm"))`

Reply:
już nie ma problemu z konwersją, teraz korzystamy z pysparkowej  funkcji do liczenia ks testu w którem nie trzeba normalizować wektora